README.ci

```diff
@@ -75,3 +76,29 @@ The `.overlay` file will contain a set of variables that will be used
 to do the corresponding `opam pin` or to overload the corresponding
 git repositories, etc...
 
+Since pull requests only happen on Github there is no need to test the
+corresponding Gitlab CI variables.
+
+Travis specific information
+===========================
+
+Travis rebuilds all of Coq's executables and stdlib for each job. Coq
+is built with `./configure -local`, then used for the job's test.
+
+Gitlab specific information
+===========================
+
+Gitlab is set up to use the "build artifact" feature to avoid
+rebuilding Coq. In one job, Coq is built with `./configure -prefix
+install` and `make install` is run, then the `install` directory
+persists to and is used by the next jobs.
```

Comment:
persists to? I'm not sure the "to" is necessary here or something is missing. (Although I'm not a native English speaker so if you tell me this is fine, I cannot argue.)